src/index.js

```diff
@@ -74,3 +76,20 @@ todoContainer.addEventListener("click", (e) => {
     populateList(getToDos());
   });
 });
+
+let indexesToRemove = [];
+
+todoContainer.addEventListener("click", (e) => {
+  const activeEl = e.target.closest(".checkbox");
+  if (!activeEl) return;
+  const parentEl = activeEl.closest(".todo");
+
+  indexesToRemove = checkoffItem(parentEl);
+});
```

Comment:
- [ ] Kindly notice that you should update the `localstorage` data when the status change.

Reply:
@ShahierNashaat  Thank you for the feedback. Please notice, based on the structure of my code, I am effectively saving on the local storage on the line below. Refer to line no. 92 in the code snippet below:
https://github.com/Serengia/to-do-app-with-webpack/blob/78c851259300d7e56f7590d1da801dcaea867eb6/src/index.js#L79-L95

Reply:
Hi, @Serengia What I mean is the status of the task from `completed: false` to `completed: true` should be updated in the local storage, as when I marked the task as completed and refreshed the page the task returned to not completed status.